docker/Dockerfile

```diff
@@ -0,0 +1,40 @@
+ARG CUDA_VERSION=10.2
```

Comment:
This shouldn't matter, since pytorch also comes with precompiled cuda. I chose that, since this is the version that may be used, if somebody wants to compile custom stuff.

Reply:
It will be important for Horovod and I believe Apex/AMP, as they both compile from CUDA source.